pom.xml

```diff
@@ -0,0 +1,258 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>org.springframework.boot</groupId>
+    <artifactId>spring-boot-starter-parent</artifactId>
+    <version>2.3.4.RELEASE</version>
+    <relativePath/>
+  </parent>
+
+  <groupId>org.folio</groupId>
+  <artifactId>edge-search-srs</artifactId>
+  <version>1.0-SNAPSHOT</version>
+
```

Comment:
This version number is broken, should be x.y.z-SNAPSHOT

Is there some Spring Way template that has a broken POM? We saw this same error with mod-bursar-export [FOLIO-3001](https://issues.folio.org/browse/FOLIO-3001}

Reply:
This version is specified when creating a project. Fixed.